src/jit/codegenarm.cpp

```diff
@@ -19,6 +19,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 #ifdef _TARGET_ARM_
 #include "codegen.h"
+#include "sideeffects.h"
```

Comment:
Why is this included here? If it is a dependency of one of the other include files, shouldn't it be included there?